src/test/java/lotto/domain/LottoTest.java

```diff
@@ -60,12 +59,12 @@ class LottoTest {
 
     private static List<Arguments> LottoMatchParameterProvider() {
         return List.of(
-                Arguments.of(Arrays.asList(1, 3, 5, 7, 9, 11), LottoResult.THREE),
-                Arguments.of(Arrays.asList(1, 2, 3, 4, 9, 11), LottoResult.FOUR),
-                Arguments.of(Arrays.asList(1, 2, 3, 4, 5, 11), LottoResult.FIVE),
-                Arguments.of(Arrays.asList(1, 2, 3, 4, 5, 7), LottoResult.BONUS),
-                Arguments.of(Arrays.asList(1, 2, 3, 4, 5, 6), LottoResult.SIX),
-                Arguments.of(Arrays.asList(6, 7, 8, 9, 10, 11), LottoResult.FAIL)
+                Arguments.of(new Integer[]{1, 3, 5, 7, 9, 11}, LottoResult.THREE),
```

Comment:
하단과 같이 주생성자를 이용하게 해볼수도 있을것 같습니다!

```suggestion
                Arguments.of(Set.of(1, 3, 5, 7, 9, 11)}, LottoResult.THREE)
```